SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialPlayerSpawner.cpp

```diff
@@ -68,10 +68,10 @@ void USpatialPlayerSpawner::SendPlayerSpawnRequest()
 		{
 			checkf(Op.result_count == 1, TEXT("There should never be more than one SpatialSpawner entity."));
 
-			SpatialGDK::SpawnPlayerRequest SpawnRequest = ObtainPlayerParams();
+			SpawnPlayerRequest SpawnRequest = ObtainPlayerParams();
 			Worker_CommandRequest SpawnPlayerCommandRequest = PlayerSpawner::CreatePlayerSpawnRequest(SpawnRequest);
 			NetDriver->Connection->SendCommandRequest(Op.results[0].entity_id, &SpawnPlayerCommandRequest,
-													  SpatialConstants::PLAYER_SPAWNER_SPAWN_PLAYER_COMMAND_ID);
+													  SpatialConstants::PLAYER_SPAWNER_SPAWN_PLAYER_COMMAND_ID, RETRY_UNTIL_COMPLETE, {});
```

Comment:
As this previously only tried MAX_NUMBER_COMMAND_ATTEMPTS times should we keep that or is this one better if it keeps retrying forever?